src/repos/profile.js

```diff
@@ -8,6 +8,7 @@ function getProfile(db, steamId) {
     COALESCE(profile.name, steam_user.name) AS name,
     steam_user.name AS steam_name,
     profile.faceit_name as faceit_name,
+    profile.discord_name as discord_name,
```

Comment:
Was this already migrated? I think I remember seeing it but just want to double-check.

Reply:
I meant `discord_name` - somehow I missed the migration at the top of the files changed, my bad.